bazel/repository_locations.bzl

```diff
@@ -601,6 +601,30 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2021-12-24",
         cpe = "N/A",
     ),
+    com_github_intel_isa_l = dict(
+        project_name = "isa-l",
+        project_desc = "Intel(R) Intelligent Storage Acceleration Library",
+        project_url = "https://github.com/intel/isa-l",
+        version = "2.30.0",
+        sha256 = "bcf592c04fdfa19e723d2adf53d3e0f4efd5b956bb618fed54a1108d76a6eb56",
+        strip_prefix = "isa-l-{version}",
+        urls = ["https://github.com/intel/isa-l/archive/v{version}.tar.gz"],
+        use_category = ["controlplane", "dataplane_core"],
+        release_date = "2020-11-12",
```

Comment:
the release date of version is different from it, see https://dev.azure.com/cncf/envoy/_build/results?buildId=114782&view=logs&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467&l=272

Reply:
OK, I got it, thanks!